pkg/apis/management.cattle.io/v3/catalog_types.go

```diff
@@ -35,11 +35,10 @@ type CatalogSpec struct {
 }
 
 type CatalogStatus struct {
-	LastRefreshTimestamp string `json:"lastRefreshTimestamp,omitempty"`
-	Commit               string `json:"commit,omitempty"`
-	// helmVersionCommits records hash of each helm template version
-	HelmVersionCommits map[string]VersionCommits `json:"helmVersionCommits,omitempty"`
-	Conditions         []CatalogCondition        `json:"conditions,omitempty"`
+	LastRefreshTimestamp string             `json:"lastRefreshTimestamp,omitempty"`
+	Commit               string             `json:"commit,omitempty"`
+	HelmVersionCommits   []byte             `json:"helmVersionCommits,omitempty"`
```

Comment:
This will need to be a new field instead and the old field will need to be deprecated